sdk/communication/azure-communication-rooms/tests/test_hmac.py

```diff
@@ -5,7 +5,6 @@
 # license information.
 # --------------------------------------------------------------------------
 
-import unittest
 from azure.communication.rooms._shared.policy import HMACCredentialsPolicy
 from devtools_testutils import AzureTestCase
```

Comment:
It looks like these tests don't use test class attributes and don't make service requests/aren't recorded. Could the `HMACTest` class drop its `AzureTestCase` inheritance? This would make it easier for us to remove `AzureTestCase` from our tooling, which we plan to do in the future 🙂

Reply:
Yes, great point. I've removed AzureTestCase inheritance :)